ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,201 @@
+the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people.
+
+Once the champion has asked the Zarr community whether an idea has any chance of acceptance, a draft ZEP should be presented to the appropriate venue mentioned 
+above. This gives the author a chance to flesh out the draft ZEP to make properly formatted, of high quality, and to address initial concerns about the proposal.
+
+After the PR for the ZEP is in place, a post should be made on the Gitter channel containing the sections up to “Backward compatibility”, to limit discussion 
+there to usage and impact. Discussion on the pull request will have a broader scope, also including details of implementation.
+
+Standards Track ZEPs consist of two parts, a design document and a reference implementation. It is generally recommended that at least a prototype implementation 
+be co-developed with the ZEP, as ideas that sound good in principle sometimes turn out to be impractical when subjected to the test of implementation. 
+Often it makes sense for the prototype implementation to be made available as PR to the Zarr repo (making sure to appropriately mark the PR as a WIP).
+
+### Submitting a ZEP
+
+The proposal should be submitted as a draft ZEP via a GitHub pull request to the zarr-developers/zep directory with the name `zep-<n>.rst` where `<n>` is an appropriately assigned four-digit number (e.g., zep-0000.rst). The draft ZEP must use the [ZEP X - Template and Instructions](https://github.com/MSanKeys963/governance/blob/master/ZEP/template/template.md) file.
+
+A few points to consider while submitting your ZEP:
+
+- It should sound complete. The ideas must make technical sense.
+- The title should accurately describe the content.
+- The ZEP’s language (spelling, grammar, sentence structure etc.) and code style should be correct and conformant.
+
+The [Zarr Steering Council](https://github.com/zarr-developers/governance/blob/master/GOVERNANCE.md#steering-council) will not unreasonably deny publication of a ZEP. Reasons for denying ZEP include duplication of effort, being technically unsound, not providing proper 
+motivation or addressing backwards compatibility, or not taking care of Zarr [CODE OF CONDUCT](https://github.com/zarr-developers/.github/blob/main/CODE_OF_CONDUCT.md).
+
+### Discussing a ZEP
+
+As soon as the draft ZEP is committed to the ZEP repository, a discussion thread for the ZEP should be created to provide a central place to discuss and review 
+its contents. The ZEP author(s) may select the [GitHub discussion feature](https://github.com/zarr-developers/zarr-python/discussions) in the zarr-python repository, 
+open a new issue in the Zarr community repository or start the discussion in the Zarr Gitter channel. The discussion regarding the ZEP should follow Zarr’s 
+[CODE OF CONDUCT](https://github.com/zarr-developers/.github/blob/main/CODE_OF_CONDUCT.md) at all times.
```

Comment:
I think we should just discuss the ZEP in the ZEP PR.

Reply:
I've added multiple avenues for discussions [here](https://github.com/zarr-developers/governance/pull/16).